spec/models/generic_file_spec.rb

```diff
@@ -744,4 +744,18 @@ def paranoid_edit_permissions
       described_class.where_public
     end
   end
+  describe "to_csv" do
+    before do
+      @file.title = ["My Title"]
+      @file.creator = ["Von, Creator"]
+      @file.resource_type = ["Book", "Other"]
+      @file.rights = ["Mine"]
+      @file.mime_type = "application/pdf"
+    end
+    subject { @file.to_csv }
+    it do
```

Comment:
I get what's going on here but having an `it do` block looks weird to me. Is this a rubocop auto-fix? I'd expect to see:

```ruby
it { is_expected.to eq 'My Title, ...' }
it 'parses into an array' do
  expect(CSV.parse(subject)).to eq [['My Title', ... ]]
end
```
(And do we expect an array there or an array of arrays?)